tests/calculations/test_pp.py

```diff
@@ -24,6 +24,8 @@ def _generate_inputs(parameters=None):
             generate_remote_data(fixture_localhost, fixture_sandbox.abspath, 'quantumespresso.pw'),
             'parameters':
             orm.Dict(dict=parameters),
+            'settings':
+            orm.Dict(dict=settings) or orm.Dict(),
```

Comment:
I don't think this works as expected. The `or` won't be applied to the value of `settings` but `orm.Dict(dict=settings)`, so no matter the value of `settings` you will always hit the first clause. Since `orm.Dict(dict=None)` is fine and is exactly equal to `orm.Dict()` you might as well get rid of `or orm.Dict()`

Reply:
ah, didn't know. And you are right, empty AiiDA `Dict` object is not falsy:
```python
In [3]: if Dict(dict={}):
   ...:     print("Hello")
   ...:
Hello
```

Maybe one should fix this here as well:
https://github.com/aiidateam/aiida-quantumespresso/blob/0031475aff4aac8677c8da31c90e5498aad24b37/tests/parsers/test_pw.py#L25

Reply:
Ah, yes, please do

Reply:
also, just a side note:
```python
In [6]: Dict(dict=None) == Dict(dict=None)
Out[6]: False
```
not sure if this is what we expect.